locales/en/yourloops.json

```diff
@@ -509,8 +509,8 @@
   "product-labelling": "Product Labelling",
   "profession-dialog-title": "Please choose a profession",
   "professional-account": "Professional account",
-  "profile-update-failed": "User account update failed, not all data could be saved",
-  "profile-updated": "User account updated",
+  "profile-update-failed": "Profile update failed, not all data could be saved",
```

Comment:
This update is wrong, the translation should be the previous one (with "User account" and not "Profie")